middlewares/validators/validator.helper.js

```diff
@@ -568,6 +568,46 @@ function enumValidator(fieldLocation, fieldname, enums, optional = true) {
     }
 }
 
+/**
+ * Validates that the field is a valid hacker update object, and checks if corresponding new status is valid.
+ * @param {"query" | "body" | "header" | "param"} fieldLocation The location where the field should be found.
+ * @param {string} actionFieldName The name of the action that needs to be performed.
+ * @param {string} statusFieldName The name of the action that needs to be performed.
+ */
+function updateHackerValidator(fieldLocation, fieldName) {
+    const hackerObjectValue = setProperValidationChainBuilder(fieldLocation, fieldName, "Invalid hacker update object string.");
+
+    return hackerObjectValue.exists()
+        .withMessage("The hacker update object string must exist.")
+        .custom(updateHackerValidatorHelper).withMessage("The value must be a valid hacker update object.");
+}
+
+function updateHackerValidatorHelper(update) {
+    try {
+        var updateObject = JSON.parse(update);
+
+        if (updateObject && typeof updateObject === "object" && !("password" in updateObject)) {
```

Comment:
Does the hacker schema have a password? If not, then we can remove the "password" part.